docs/src/routes/Routes.js

```diff
@@ -132,20 +128,8 @@ export const ROUTES_MAPPINGS = [
   {
     path: ROUTES.USING_BACKPACK,
     layout: UsingLayout,
-    redirect: ROUTES.GETTING_STARTED,
-    routes: [
-      {
-        path: ROUTES.GETTING_STARTED,
-        component: GettingStartedPage,
-      },
-      {
-        path: ROUTES.BACKPACK_REACT_SCRIPTS,
-        component: BackpackReactScriptsPage,
-      },
-      { path: ROUTES.BASE_STYLESHEET, component: BaseStylesheetPage },
-      { path: ROUTES.PRINCIPLES, component: PrinciplesPage },
-      ...autogeneratedUsingPages,
-    ],
+    redirect: autogeneratedUsingPages[0].path,
```

Comment:
I figured this was an ok way to make redirects work without having to hard code the path of an autogen file in here. It'll just redirect users to whatever the first autogen page is.